setup.py

```diff
@@ -33,7 +33,7 @@
     author="Saasha Joshi",
     author_email="saashajoshi08@gmail.com",
     url="https://github.com/SaashaJoshi/piQture",
-    packages=["piqture", "tests"],
+    packages=find_packages(include=["piqture", "piqture *", "tests"]),
```

Comment:
Hi @ketayon,
Thanks for making the changes so soon. I think it should be `piqture.*`, and the rest LGTM. 

`packages=find_packages(include=["piqture", "piqture.*", "tests"]),`